libs/ui-toolkit/src/components/healthbar/healthbar.tsx

```diff
@@ -0,0 +1,217 @@
+import classNames from 'classnames';
+import * as Schema from '@vegaprotocol/types';
```

Comment:
Toolkit package shouldn't care about what types are/are not in Vega conceptually.

Reply:
Suggest externalising this dependency by taking an intent prop

Reply:
Made this change